### PR TITLE
docs: add docs on enterprise catalog

### PR DESCRIPTION
**Pull request by takirala**
### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4184.s3-website-us-west-2.amazonaws.com/

Based on top of https://github.com/mesosphere/dcos-docs-site/pull/4193/files

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        kommander.d2iq.io/project-default-catalog-repository: "true"
+        kommander.d2iq.io/workspace-default-catalog-repository: "true"
+        kommander.d2iq.io/gitapps-gitrepository-type: "dkp"
+      gitRepositorySpec:
+        url: https://github.com/mesosphere/dkp-catalog-applications
+        ref:
+          tag: v2.2.0
+```
+
+This configuration can be used when installing or reconfiguring Kommander by passing it to the `kommander install` command:
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:33]:
```suggestion
Use this configuration when installing or reconfiguring Kommander by passing it to the `kommander install` command:
```

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+```bash
+kommander install --installer-config <config_file.yaml>
+```
+
+This is the meaning of each label
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:39]:
```suggestion
Review the meaning of each label in the following table:
```

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+| Label  |  Description |
+|---|---|
+| `kommander.d2iq.io/project-default-catalog-repository`  | Indicates this acts as a Catalog Repository in all projects  |
+| `kommander.d2iq.io/workspace-default-catalog-repository`  | Indicates this acts as a Catalog Repository in all workspaces  |
+| `kommander.d2iq.io/gitapps-gitrepository-type`  |  Indicates this Catalog Repository (and thus, all its Applications) are certified to be working on DKP platform |
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:45]:
```suggestion
| `kommander.d2iq.io/gitapps-gitrepository-type`  |  Indicates this Catalog Repository (and all its Applications) are certified to run on DKP |
```

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+When running in airgapped environments, update the above configuration by replacing `gitRepositorySpec` with `path` field pointing to a local path of the catalog git repository folder. For e.g.:
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:47]:
```suggestion
When running in air-gapped environments, update the configuration by replacing `gitRepositorySpec` with the `path` field pointing to a local path of the catalog git repository folder. E.g.:
```

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+```bash
+git clone https://github.com/mesosphere/dkp-catalog-applications
+```
+
+and then
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:53]:
```suggestion
And then:
```

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
+---
+layout: layout.pug
+navigationTitle: Configure enterprise catalog
+title: Configure enterprise catalog
+menuWeight: 30
+excerpt: Configure enterprise catalog for Kommander
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:6]:
```suggestion
excerpt: Configure an Enterprise catalog for Kommander
```

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+beta: false
+enterprise: true
+---
+
+Kommander supports configuring default catalog(s) for clusters with enterprise license.
+
+## Configure a default enterprise catalog
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:13]:
```suggestion
## Configure a default Enterprise catalog
```

**Review by gvasquezvargas — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+Review the meaning of each label:
+
+| Label  |  Description |
+|---|---|
+| `kommander.d2iq.io/project-default-catalog-repository`  | Indicates this acts as a Catalog Repository in all projects  |
+| `kommander.d2iq.io/workspace-default-catalog-repository`  | Indicates this acts as a Catalog Repository in all workspaces  |
+| `kommander.d2iq.io/gitapps-gitrepository-type`  |  Indicates this Catalog Repository (and all its Applications) are certified to run on DKP |
+
+When running in air-gapped environments, update the configuration by replacing `gitRepositorySpec` with the `path` field pointing to a local path of the catalog git repository folder, for example:
```
[the remark this answers is not in the document]
Comment (takirala) [reply pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:49]: done

File: pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+excerpt: Configure enterprise catalog for Kommander
+beta: false
+enterprise: true
+---
+
+Kommander supports configuring default catalog(s) for clusters with enterprise license.
+
+## Configure a default enterprise catalog
+
+To configure Kommander to use a default catalog repository, create the following yaml file:
```
[the remark this answers is not in the document]
Comment (takirala) [reply pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md:15]: nope.

**Review by cbuto — APPROVED** [review]

**Review by cwyl02 — APPROVED** [review]

**Comment by takirala** [comment]
cc @cmastr @msdolbey 
can a authorized user merge this PR ? TIA!

**Pull request merged**